### PR TITLE
refactored PrivateStaticMethod

### PR DESCRIPTION
**Pull request by Vitaly-Protasov**
Solved in case of #528

### CODE REVIEW

File: aibolit/patterns/private_static_method/private_static_method.py [hunk 1]
```diff
@@ -1,16 +1,44 @@
...
 
 
 class PrivateStaticMethod:
+    '''
+    Once you see a 'private static' method, it's a pattern.
+    '''
+    def _check_private_static(self, node: ASTNode) -> bool:
+        if all(type in node.modifiers for type in ['private', 'static']):
+            return True
+        return False
```
Comment (aravij) [inline aibolit/patterns/private_static_method/private_static_method.py:35]:
```suggestion
        return {'private', 'static'} <= node.modifiers
```

Comment (Vitaly-Protasov) [reply aibolit/patterns/private_static_method/private_static_method.py]: solved

**Comment by acheshkov** [comment]
@rultor merge

**Comment by acheshkov** [comment]
@rultor merge

**Pull request merged**